src/stories/components/WebcamControls/WebcamControls.js

```diff
@@ -0,0 +1,68 @@
+import './styles.scss';
+
+import React, { useState } from 'react';
+import Webcam from 'react-webcam';
+
+import { Constrain } from '../../layouts/Constrain/Constrain';
+import { Button } from '../Button/Button';
+import { ImageGuidelines } from '../ImageGuidelines/ImageGuidelines';
+import { ToggleCamera } from '../ToggleCamera/ToggleCamera';
+
+/**
+ * Component for webcam controls element.
+ *
+ * @component
+ * @return {object} (
+ *   <WebcamControls />
+ * )
+ */
+export const WebcamControls = () => {
+  const [cameraIsOn, setWebcamOn] = useState(false);
+  const [snapshot, setSnapshot] = useState('');
+  const webcamRef = React.useRef(null);
+
+  // toggle device camera
+  const toggleCamera = () => {
+    setWebcamOn(!cameraIsOn);
+  };
+
+  // take a photo
+  const capturePhoto = React.useCallback(
+    () => {
+      setSnapshot(webcamRef.current.getScreenshot());
```

Comment:
I am surprised at how quickly we are able to capture the image and present it! That's awesome :D.